xapian-core/tests/api_cluster.cc

```diff
@@ -80,3 +105,35 @@ DEFINE_TESTCASE(round_robin1, backend)
     }
     return true;
 }
+
+DEFINE_TESTCASE(stem_stopper1, backend)
+{
+    Xapian::Stem stemmer("english");
+    // By default, stemming strategy used is STEM_SOME
+    Xapian::StemStopper stopper(stemmer);
+    std::string term = "the";
+    stopper.add(term);
+    TEST(stopper(term));
+    TEST(stopper('Z' + stemmer(term)));
+
+    Xapian::StemStopper stopper_all_z(stemmer, Xapian::StemStopper::STEM_ALL_Z);
+    Xapian::StemStopper stopper_all(stemmer, Xapian::StemStopper::STEM_ALL);
+    term = "because";
+    stopper_all.add(term);
+    stopper_all_z.add(term);
+    TEST(!stopper_all_z(term));
+    TEST(stopper_all_z('Z' + stemmer(term)));
+    TEST(!stopper_all(term));
+    TEST(stopper_all('Z' + stemmer(term)));
```

Comment:
I just checked out your branch, and indeed this check fails for me.

You really should be running `make check` before pushing changes.